src/test/scala/org/broadinstitute/dsde/workbench/leonardo/api/LeoRoutesSpec.scala

```diff
@@ -240,6 +240,26 @@ class LeoRoutesSpec extends FlatSpec with ScalatestRouteTest with TestLeoRoutes
     }
   }
 
+  it should "create a cluster with stopAfterCreation specified" in {
```

Comment:
Added these unit tests for coverage but as mentioned above it doesn't really catch the bug since the same model/JSON serializer is used in the tests and application.